src/common/core.py

```diff
@@ -449,7 +449,7 @@ def get_unit_name(class_name, unit_type='FMU'):
             The name of the model.
             
         unit_type --
```

Comment:
It looks like one could actually remove the whole `get_unit_name `function, couldn't find any place it is used.